src/box/tuple.h

```diff
@@ -327,13 +334,12 @@ tuple_data_range(const struct tuple *tuple, uint32_t *p_size)
  * @retval NULL     Memory allocation error
  */
 char *
-tuple_extract_key(const struct tuple *tuple, const struct key_def *key_def,
-		  uint32_t *key_size);
-
+tuple_extract_key(const struct tuple *tuple,
+		const struct key_def *key_def, uint32_t *key_size);
 /**
- * Extract key from raw msgpuck by given key definition and return
+ * Function extracts key from raw msgpuck by given key definition and return
```

Comment:
No, please, leave the previous version. Stop increasing diff by unneccessary changes.